src/api/commentType.tsx

```diff
@@ -0,0 +1,64 @@
+interface comment {
+  commentId?: number;
+  postId: number;
+  userId: string;
+  description: string;
+  postedAt: number;
+  lastUpdatedAt: number;
+}
+
+interface fetchPostCommentsApiPayload {
```

Comment:
for types or interfaces used for API payloads, i recommend doing this

Create a new `types.tsx` file for `/api` for types
Put this inside
```
export interface Payload<T> {
  payload: T;
}
```

Then when you want to create payload types in any feature files for types, you can do this
```
interface FetchPostComments {
    postId: number
}

export type FetchPostCommentsPayload = Payload<FetchPostComments>
```

Much more concise this way